pkgs/development/python-modules/bytewax/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, rustPlatform
+, setuptools-rust
+, openssl
+, pkg-config
+, cyrus_sasl
+, protobuf
+, cmake
+, gcc
+, dill
+, multiprocess
+, pytestCheckHook
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "bytewax";
+  version = "0.15.1";
+  format = "pyproject";
+
+  disabled = pythonAtLeast "3.11";
+
+  src = fetchFromGitHub {
+    owner = "bytewax";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-E11IjE5cThVG88ZcpWSgrdjkkHgU05oyT4affEeXCIo=";
```

Comment:
```suggestion
    hash = "sha256-4HZUu3WSrhxusvuVz8+8mndTu/9DML1tCH52eaWy+oE=";
```

Reply:
@SuperSandro2000 Why has the hash been updated?

Reply:
because ofborg fails and I took the hash from the logs. You maybe pasted a wrong one or didn't update it after testing or so.

Reply:
I've updated the hash, it seems to build with either hash when I run it on my machine, so a bit confused as to why the ofborg failed though.

Reply:
you have both locally cached, so thats why it works. Try collecting garbage and then you should only get the one from ofborg.